docker/caddy/Caddyfile

```diff
@@ -0,0 +1,14 @@
+http://grassroots.local {
```

Comment:
In case you're curious, this isn't being used at all right now, but I expect we'll need it for getting auth working consistently between local dev and a hypothetical production environment.